sonar-orchestrator/src/test/java/com/sonar/orchestrator/OrchestratorBuilderTest.java

```diff
@@ -63,13 +63,13 @@ public void install_plugins_on_sonarqube_lts() throws Exception {
     Orchestrator orchestrator = new OrchestratorBuilder(Configuration.create())
       .setSonarVersion(LTS_ALIAS)
       // fixed version
-      .addPlugin(MavenLocation.of("org.sonarsource.xml", "sonar-xml-plugin", "1.5.0.1373"))
+      .addPlugin(MavenLocation.of("org.sonarsource.xml", "sonar-xml-plugin", "2.0.1.2020"))
```

Comment:
Is the old LTS using this version? do we know why it was hardcoded to 1.5.0.1373?

Reply:
No idea, it was probably among the newest when the test was written. I tracked the issue into sonar-analyzer-common that used the JDK implementation of parser until 2018 when this implementation was removed from jdk17.

Reply:
Just checked and 8.9 is using version 2.2.0.2973 of the plugin. So this should not be an issue.